lib/interpreters/periodic_visits.rb

```diff
@@ -64,11 +64,15 @@ def expand(vrp, job, &block)
       vrp
     end
 
-    def generate_timewindows(timewindows_set)
+    def generate_timewindows(timewindows_set, only_day_index = nil)
```

Comment:
From my point of view the prefix `only_` assumes that the variable is a boolean
```suggestion
    def generate_timewindows(timewindows_set, day_index = nil)
```